docs/UserGuide.md

```diff
@@ -90,42 +90,56 @@ Shows a message explaning how to access the help page.
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding a volunteer: `add_volunteer`
 
-Adds a person to the address book.
+Adds a volunteer to the address book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `add_volunteer n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS ag/AGE vnr/NRIC [t/TAG]…​`
+* Every volunteer must have a unique `NRIC`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
+A volunteer can have any number of tags (including 0)
 </div>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add_volunteer n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 ag/23 vnr/S8457677H`
+* `add_volunteer n/Betsy Crowe t/graduate e/betsycrowe@example.com a/Newgate Prison p/1234567 ag/27 vnr/S8959886I t/experienced`
 
-### Listing all persons : `list`
+### Listing all persons : `list_volunteer`
 
-Shows a list of all persons in the address book.
 
-Format: `list`
+Shows a list of all volunteers in the application.
 
-### Editing a person : `edit`
 
-Edits an existing person in the address book.
+Format: `list_volunteer`
 
```

Comment:
When I try out the newest version of the code, when adding an elderly or a volunteer, the newly added records is no longer displayed in the scrolling view of the UI. Just curious if this is expected for v1.2?

Reply:
This is expected for v1.2a (which is going to end by this week! Adding an `elderly` and a `volunteer` will add it to the `UniqueElderlyList` and `UniqueVolunteerList` respectively, but these lists are not yet shown on the UI. We will implement this moving into v1.2b and v1.2c!

Reply:
I believe @Zxun2 is in the middle of abstracting out the person list into volunteer list and elderlylist. The new volunteers are added into volunteer list instead of unique person list, which is why it does not show up in unique person list in the UI